specs/differences-from-optimism-bedrock.md

```diff
@@ -23,33 +23,46 @@
 
 ## Nodes
 
-### Overview
+There are two types of network participants in the OP Stack:
+* [[Sequencers|https://github.com/ethereum-optimism/optimism/blob/develop/specs/introduction.md#sequencers]] consolidate users' on/off chain transactions into blocks. They submit checkpoint outputs as well as batch transactions.
```

Comment:
Please put blank here! (Please run the markdown lint!)

Reply:
long lines are separated into proper length lines.